README.md

```diff
@@ -20,6 +20,7 @@
 - [GitHub Readme Twitter](https://github.com/gazf/github-readme-twitter#readme) - Add Twitter to your github readmes.
 - [Profile Activity Generator](https://github.com/omidnikrah/profile-activity-generator#readme) - Generate custom profile activity for your profile readme.
 - [Readme Pagespeed Insights](https://github.com/ankurparihar/readme-pagespeed-insights#readme) - Google lighthouse stats of your website that you can put in readme.
+- [Spotify Github Profile](https://github.com/kittinan/spotify-github-profile) - Show your Spotify playing on your Github profile.
```

Comment:
This widget already exists in Miscellaneous section.
What you should do is first check whether the widget already exists or not before opening a PR request.

Reply:
my bad :(